cicecore/cicedyn/infrastructure/ice_grid.F90

```diff
@@ -1489,6 +1489,22 @@ subroutine rectgrid
             enddo
             enddo
 
+         elseif (trim(kmt_type) == 'channel_oneeast') then
+
+            do j = ny_global/2,ny_global/2    ! one channel wide
```

Comment:
what happens if ny_global is odd?

Reply:
It doesn't matter, it'll just round down.  ny_global/2 is always a well defined integer value.  The channel will be created at j=ny_global/2, whether ny_global is odd or even.  We could have chosen j=2 or j=ny_global/4 or anything really would work.  ny_global/2 is probably safest.  If ny_global is a relatively small integer, it'll create the channel away from the edge as much as possible.  I guess we run into a problem if ny_global is less than 4, but that's not a particularly useful grid, and I don't think we need to worry about that.